### DISCUSSION TITLE
Sansa Review

### DISCUSSION THREAD

**Discussion by fanevanjanahary** [kartoza/catalogue#671] [General]
@Maite-Machipi please can you review the order form and the admin page. Thank you

**Comment by Maite-Marageni** [comment]
@Fanevanjanahary  please note that im still getting the 502 bad gateway when trying to access the admin page im also not able to view the order form i get 500: There was a problem with this request :-(

**Reply by fanevanjanahary** [reply]
Hi @Maite-Machipi can you please check again?

**Reply by Maite-Marageni** [reply]
Hi @Fanevanjanahary  im still getting the 502 bad gateway error.

**Reply by fanevanjanahary** [reply]
Hello, @Maite-Machipi Can you please confirm if you have access to the order form? I was asking a colleague to test, and he didn't get any error neither did I.

**Reply by Maite-Marageni** [reply]
Hi @Fanevanjanahary  I can now access the order from when trying to place an order, but I see the use of data doesn't have the dropdown list.

Please note that I'm still not able to access the admin page.

**Reply by fanevanjanahary** [reply]
Hello @Maite-Machipi, please can you check the admin page, it works on my side 
![image](https://user-images.githubusercontent.com/12623986/169251932-f888a89d-747f-4eae-a74d-490555653586.png)

**Reply by Maite-Marageni** [reply]
![image](https://user-images.githubusercontent.com/15606838/169268374-6ca9f55c-bc85-4033-8cf4-46321509d9e8.png)
@Fanevanjanahary  please note that im still getting the error on my side, i will ask someone else to test it.
